partcad/tests/unit/test_part.py

```
@@ -70,6 +70,26 @@ def test_part_get_brep_2():
+def test_part_get_obj_1():
+    """Load a OBJ part from a project by the part name"""
+    ctx = pc.Context("examples/produce_part_obj")
+    repo1 = ctx.get_project(".")
+    assert repo1 is not None
+    box = repo1.get_part("cube")
+    assert box is not None
+    wrapped = asyncio.run(box.get_wrapped(ctx))
+    assert wrapped is not None
+
+
+def test_part_get_obj_2():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_part_get_obj_2`

Add return type annotation: `None`

(ANN201)